client/scala/src/main/scala/ai/verta/blobs/dataset/Dataset.scala

```diff
@@ -64,7 +64,7 @@ object Dataset {
      component.last_modified_at_source.get,
      component.md5.get,
```

Comment:
More of these fields can be empty. I recommend using getOrElse in most places for safety.

Reply:
I'll use 0 for int values, and empty string for hash. Sounds good?